### PR TITLE
🔭 [Frontend/ProductCatalogService] Updated Product Catalog Service Data to Match Astronomy Store

### PR DESCRIPTION
**Pull request by xoscar**
Fixes https://github.com/open-telemetry/opentelemetry-demo/issues/282.

## Changes

Updates the product service catalog data to have products that match the OTEL astronomy theme.

https://www.loom.com/share/828a3e2e6b1e4802894abeb3deda9464

### CODE REVIEW

**Review by reyang — CHANGES_REQUESTED** [review]
Some of the images could have legal/copyright concerns, e.g. https://raw.githubusercontent.com/open-telemetry/opentelemetry-demo/e55ee5f4e734e38e5ad1a7dd6713509d70da63b9/src/frontend/public/products/LensCleaningKit.webp seems to be coming from https://www.amazon.com/Celestron-93576-Lens-Cleaning-Kit/dp/B08N5D9LXH.

**Comment by mic-max** [comment]
@xoscar Could you run the local service tests to make sure nothing broke there `/test/`
Also, good idea to keep the existing `ID`
I agree with @reyang though, we should source all free to use images

**Review by mic-max — CHANGES_REQUESTED** [review]
@xoscar Could you run the local service tests to make sure nothing broke there `/test/`
Also, good idea to keep the existing `ID`
I agree with @reyang though, we should source all free to use images

**Comment by xoscar** [comment]
Hey @mic-max  & @reyang thanks for your review. I'm getting the non-copyrighted images and I'll be updating the PR.

For the /tests/ I found out all of the tests are failing because of this change https://github.com/open-telemetry/opentelemetry-demo/commit/55d9919dde029a0d57a947cdecb157faae7844ff I think we should run the tests within the containers context so we don't have to expose the ports to localhost.

**Comment by mic-max** [comment]
> context so we don't have to expose the ports to localhost
Ah yes, I forgot about that PR. I will work on getting those tests back up and running :)

**Comment by xoscar** [comment]
@mic-max  & @reyang the PR should be ready for a second review 😬

File: src/adservice/src/main/java/hipstershop/AdService.java [hunk 1]
```diff
@@ -181,48 +181,47 @@ private void blockUntilShutdown() throws InterruptedException {
   }
 
   private static ImmutableListMultimap<String, Ad> createAdsMap() {
-    Ad hairdryer =
```
Comment (xoscar) [inline src/adservice/src/main/java/hipstershop/AdService.java:184]: Updating the ads to match the new products

**Comment by xoscar** [comment]
Hey @mic-max  & @reyang let me know if there is anything else I can do. The PR should be ready for its final review 😄

**Comment by puckpuck** [comment]
Root spans for traces initiated from the browser are missing.
Screen Shot 2022-08-15 at 11 38 52 PM

Using chrome dev tools, you can see 405 errors to send a payload of what seems to be an OTLP JSON payload.

Screen Shot 2022-08-15 at 11 54 10 PM
Screen Shot 2022-08-15 at 11 54 39 PM

**Review by mic-max — APPROVED** [review]
Looks great!

**Comment by cartersocha** [comment]
Could we get a changelog entry and potentially readme guidance on how to add more items? The readme update can potentially follow in another pr

**Comment by cartersocha** [comment]
We should agree on some sort of file naming conventions to follow too. Our docs are word_word and pictures are word-word. These new images are WordWord.

**Comment by cartersocha** [comment]
I'm fine with pictures all one way and markdown files the other but we need some sort of consistency here

**Review by puckpuck — APPROVED** [review]

**Comment by puckpuck** [comment]
Did we address @reyang's concern about ensuring all images are not copyrighted? If so, we can resolve that conversation and merge this.

**Comment by xoscar** [comment]
I just added a changelog entry and I think it should be ready to be merged. @cartersocha I'll add the steps to add a new product on a separate PR if you don't mind. CC: @puckpuck

**Comment by xoscar** [comment]
@cartersocha the copyright issue has been solved as well

**Review by reyang — APPROVED** [review]

**Comment by cartersocha** [comment]
@xoscar update & we can merge!

**Comment by xoscar** [comment]
@cartersocha done 😄

**Pull request merged**